### PR TITLE
Make Package.Open default fileaccess Read only

### PR DESCRIPTION
**Pull request by ianhays**
brings the behavior of Package.Open(Stream) in line with deskop

resolves https://github.com/dotnet/corefx/issues/9932

@stephentoub @EricWhiteDev @svick

### CODE REVIEW

File: src/System.IO.Packaging/src/System/IO/Packaging/Package.cs [hunk 1]
```diff
@@ -1176,6 +1176,7 @@ private PackageRelationshipCollection GetRelationshipsHelper(string filterString
         private static readonly FileShare s_defaultFileShare = FileShare.None;
 
         private static readonly FileMode s_defaultStreamMode = FileMode.Open;
+        private static readonly FileAccess s_defaultStreamAccess = FileAccess.Read;
```
Comment (stephentoub) [inline src/System.IO.Packaging/src/System/IO/Packaging/Package.cs]: Can this (and the other s_default*s) be const instead of static readonly?

**Comment by stephentoub** [comment]
Is there a test we can/should add?

Otherwise, LGTM.

File: src/System.IO.Packaging/src/System/IO/Packaging/Package.cs (hunk 1 above)
Comment (ianhays) [inline src/System.IO.Packaging/src/System/IO/Packaging/Package.cs]: Sure.

**Comment by ianhays** [comment]
>Is there a test we can/should add?

Yes, good idea. I added one.

**Comment by stephentoub** [comment]
Additional commits LGTM.

**Pull request merged**